src/Components/LikedSongs/LikedSongs.jsx

```diff
@@ -0,0 +1,102 @@
+import axios from "axios";
+import React, { Component } from "react";
+import { Table } from "../Table/table";
+import likedSongsImage from '../../images/liked-songs-300.png'
+import "./likedsongs.css";
+
+class LikedSongs extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            id: "",
+            access_token: "",
+            playlist: null,
+            loading: true,
+        };
+    }
+
+    componentDidMount() {
+        const id = new URLSearchParams(window.location.search).get("id");
+        const access_token = window.localStorage.getItem("access_token");
+        this.setState({
+            id: id,
+            access_token: access_token,
+            playlist: null,
+        });
+
+        const URL_ALBUM = `https://api.spotify.com/v1/me/tracks`;
+
+        axios
+            .get(URL_ALBUM, {
+                headers: {
+                    Authorization: "Bearer " + access_token,
+                    "Content-Type": "application/json",
+                },
+                params: {
+                    limit: 25
```

Comment:
Why do we limit the liked songs they get to 25?

Reply:
because when I didn't it was default to 20 and I don't know how to get all